sale_order_net_price_control/models/sale_order_line.py

```diff
@@ -0,0 +1,53 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, _, fields
+from odoo.exceptions import ValidationError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    offer_price = fields.Boolean()
+    admits_offer_prices = fields.Boolean(related='product_template_id.admits_offer_prices', store=True)
```

Comment:
Better refer this field related to final product variant (every product template field is automatically accessed by their variants, without any problem)
```suggestion
    admits_offer_prices = fields.Boolean(related='product_id.admits_offer_prices', store=True)
```